### PR TITLE
Milestone 6: Database diagram

### PR DESCRIPTION
**Pull request by Xghayor**
In this milestone we completed:
1.Added schema based on database diagram

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  CONSTRAINT invoice_id_fk FOREIGN KEY(invoice_id) REFERENCES invoices(id),
+  CONSTRAINT treatment_id_fk FOREIGN KEY(treatment_id) REFERENCES treatments(id)
+);
+-- Craete a Many to many table between medical_histories and treatments
+CREATE medical_treatments (
+  medical_histories_id INT,
+  treatments_id INT,
+  CONSTRAINT medical_histories_id_fk FOREIGN KEY(medical_histories_id) REFERENCES medical_histories(id),
+  CONSTRAINT treatment_id_fk FOREIGN KEY(treatments_id) REFERENCES treatments(id)
+);
```
Comment (thecodechaser) [inline schema_based_on_diagram.sql]: - [x] You missed the keyword `TABLE` in the above query to create the table so this query will not work as expected. Please make sure to fix this issue.

Comment (thecodechaser) [inline schema_based_on_diagram.sql]: - [x] Please make sure all FOREIGN KEYS have indexes created as requested in the project requirements.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Xghayor, and @vikas924👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Good job with database schema. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aamirkhan2478 — APPROVED** [review]
Hi @Xghayor & @vikas924, 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**